clippy_config/src/lib.rs

```diff
@@ -25,5 +25,5 @@ mod conf;
 mod metadata;
 pub mod types;
 
-pub use conf::{Conf, get_configuration_metadata, lookup_conf_file, sanitize_explanation};
-pub use metadata::ClippyConfiguration;
+pub use crate::conf::{Conf, get_configuration_metadata, lookup_conf_file, sanitize_explanation};
```

Comment:
Unless we apply the lint to Clippy sources, it should not be part of this PR as it is not necessary to implement it.

Reply:
This is part of dogfood

Reply:
This *was* the result of dogfood when the lint was not allow by default. Now that it is, this would not fail dogfood if you remove this change.

Reply:
(it is not a big deal, it just isn't necessary for this PR)